### PR TITLE
fixed charts and time output

### PR DESCRIPTION
**Pull request by lukaszgancarz98**

### CODE REVIEW

File: django_react_proj/users-fe/src/components/Home/Home.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const Home = () => {
   useEffect(() => {
     getUsers();
   }, []);
-
+  console.log(users);
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/Home/Home.js:39]: Unnecessary console statement
Consider adding eslint rule `no-console`

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js [hunk 2]
```diff
@@ -1,59 +1,55 @@
...
-  useEffect(() => {
-    const handle = setInterval(getUsers, 1000);
-    return () => clearInterval(handle);
-  }, []);
+  let done = 0;
+  let notdone = 0;
 
   if (users) {
-    users.forEach((user) => {
+    Array.prototype.forEach.call(users, (user) => {
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/SummingTime/SummingTime.js:13]: Why using prototype over previous `forEach` ?

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js [hunk 3]
```diff
@@ -1,59 +1,55 @@
...
-  const getUsers = () => {
-    axios.get(API_URL).then((res) => setUsers(res.data));
-  };
-
-  useEffect(() => {
-    const handle = setInterval(getUsers, 1000);
-    return () => clearInterval(handle);
-  }, []);
+  let done = 0;
+  let notdone = 0;
```
Comment (karolszymonczyk) [inline django_react_proj/users-fe/src/components/SummingTime/SummingTime.js:10]: use camelCase variable name like `notDone`

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js [hunk 4]
```diff
@@ -1,59 +1,55 @@
...
-      value: 3,
+      value: done,
     },
     {
       type: 'Task Not Done',
-      value: 4,
+      value: notdone,
     },
   ];
   const config = {
```
Comment (karolszymonczyk) [inline django_react_proj/users-fe/src/components/SummingTime/SummingTime.js:44]: Maybe extract configured Pie to separate component?

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js [hunk 5]
```diff
@@ -1,59 +1,55 @@
...
+    Array.prototype.forEach.call(users, (user) => {
       const uptimes = new Date(user.endTime).getTime();
       const lowtimes = new Date(user.startTime).getTime();
       const sub = uptimes - lowtimes;
       const minutes = sub / 60000;
       timevalue += minutes;
     });
   }
+  if (users) {
+    Array.prototype.forEach.call(users, (user) => {
```
Comment (karolszymonczyk) [inline django_react_proj/users-fe/src/components/SummingTime/SummingTime.js:22]: you can count `done` task using filter method like `users.filter(user => user.done).length`

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js (hunk 2 above)
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/SummingTime/SummingTime.js:13]: Changed

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js (hunk 3 above)
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/SummingTime/SummingTime.js:10]: Changed

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js (hunk 5 above)
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/SummingTime/SummingTime.js]: Done

File: django_react_proj/users-fe/src/components/SummingTime/SummingTime.js (hunk 4 above)
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/SummingTime/SummingTime.js]: Done

**Review by karolszymonczyk — APPROVED** [review]

**Pull request merged**